pumps/mongo.go

```diff
@@ -15,7 +15,7 @@ import (
 	"github.com/TykTechnologies/tyk-pump/analytics"
 )
 
-const tenMB int = 10485760
+const tenMB int = 10 << 20
```

Comment:
Is this just a readability preference?

Reply:
For me is less error prone, very fast and explicit - rather than an arbitrary number. If others have objections, will revert that bit.

Reply:
For the philistines like me, I don't know what this does, add a comment to explain?